software/controller/lib/hal/hal_stm32.cpp

```diff
@@ -125,7 +125,7 @@ void HalApi::Init() {
   /// \TODO: ensure CPUFrequency is a multiple of 10 MHz
   SystemTimer::singleton().initialize(CPUFrequency);
   InitUARTs();
-  I2C::initialize();
+  i2c1.Initialize(I2C::Speed::Fast, GPIO::Port::B, 8, 9, GPIO::AlternativeFunction::AF4);
```

Comment:
comment meanings of `8` and `9`

Reply:
sdl / sda pins of the I2C port, per [PCBsp] (comments added)